core/textile/textile.go

```diff
@@ -60,12 +60,17 @@ type Bucket interface {
 		ctx context.Context,
 		path string,
 	) (path.Resolved, error)
+	ItemsCount(ctx context.Context, path string) (int32, error)
 }
 
 type backuper interface {
 	BackupBucket(ctx context.Context, bucket Bucket) (int, error)
 	BackupFileWithReader(ctx context.Context, bucket Bucket, path string, reader io.Reader) error
 	UnbackupBucket(ctx context.Context, bucket Bucket) (int, error)
+	UnbackupFile(ctx context.Context, bucket Bucket, path string) error
```

Comment:
I removed the `backuper` interface in the sync PR since now Space Service doesn't need to know about syncing ops. The `UnbackupFile` can now be under the `RemoveItem` task executioner. For the rest maybe it makes sense to have a more granular state for the backup state of each file as we talked about in our sync.